modules/site-health/audit-full-page-cache/load.php

```diff
@@ -0,0 +1,174 @@
+<?php
+/**
+ * Module Name: Audit Full Page Cache
+ * Description: Adds a check for full page cache in Site Health status.
+ * Experimental: No
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * Require helper functions.
+ */
+require_once __DIR__ . '/helper.php';
+
+/**
+ * Adds tests to site health.
+ *
+ * @since n.e.x.t
+ *
+ * @param array $tests Site Health Tests.
+ * @return array
```

Comment:
Please always provide a description for any `@param` or `@return`.
```suggestion
 * @return array Modified Site Health tests including Page caching test.
```